openassessment/xblock/test/test_studio.py

```diff
@@ -154,6 +155,25 @@ def test_update_editor_context_saves_assessment_order(self, xblock):
         self.assertEqual(xblock.editor_assessments_order, data['editor_assessments_order'])
 
     @scenario('data/basic_scenario.xml')
+    def test_editor_context_adds_staff_in_order(self, xblock):
+        # Mock loading an old problem that has saved an editor_assessments_order without staff-assessment
+        # Note self, then peer - this deliberately does *not* match the order given in basic_scenario
+        xblock.editor_assessments_order = ['student-training', 'self-assessment', 'peer-assessment']
```

Comment:
This is the test that fails on current `ora-staff-grading` code (checkout this branch, then `HEAD~1` to verify). It simulates loading an old xblock from xml with an `editor_assessments_order` that does not match the assessments as they are defined on the problem.